airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java

```diff
@@ -229,7 +233,7 @@ public void attemptFailure(final AttemptFailureInput input) {
       emitJobIdToReleaseStagesMetric(OssMetricsRegistry.ATTEMPT_FAILED_BY_RELEASE_STAGE, jobId);
       for (final FailureReason reason : failureSummary.getFailures()) {
         MetricClientFactory.getMetricClient().count(OssMetricsRegistry.ATTEMPT_FAILED_BY_FAILURE_ORIGIN, 1,
-            MetricTags.getFailureOrigin(reason.getFailureOrigin()));
+            new MetricAttribute("failure_origin", MetricTags.getFailureOrigin(reason.getFailureOrigin())));
```

Comment:
we can use the https://github.com/airbytehq/airbyte/blob/master/airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricTags.java#L17 to avoid having small duplicated strings over the codebase.